src/parser_json.c

```diff
@@ -452,6 +452,29 @@ json_get_value(struct lyd_node_leaf_list *leaf, struct lyd_node **first_sibling,
         if (data[len++] != '[') {
             LOGVAL(ctx, LYE_XML_INVAL, LY_VLOG_LYD, leaf, "JSON data (expected begin-array)");
             return 0;
+        } else {
+            unsigned int len2 = len;
+            // Strip whitespaces
```

Comment:
doc => "Strip whitespaces" => "Skip whitespace"
Strip implies removing something, but you are just moving past the whitespace.